md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ext/HiveTableMigrationAPI.java

```diff
@@ -0,0 +1,369 @@
+                 throw new ETLException(e);
+             }
+             LOGGER.info("Connection successful");
+         }
+         return connection;
+     }
+
+     public static void closeResultset(ResultSet resultSet) throws SQLException {
+         if (resultSet != null)
+             resultSet.close();
+     }
+
+     public static void closeConnection() throws SQLException {
+         if (connection != null)
+             connection.close();
+     }
+
+     //Fetching all the tables from connected database
+     @RequestMapping(value = "/tables", method = {RequestMethod.GET})
+     @ResponseBody
+     public RestWrapperOptions getTableList(@RequestParam Map<String, String> map
+     ) {
+         String dbName = null;
+         for (String string : map.keySet()) {
+             LOGGER.info("String is" + string);
+             if (map.get(string) == null || ("").equals(map.get(string))) {
+                 continue;
+             }
+             if (string.startsWith("srcDB_")) {
+                 dbName = map.get(string);
+             }
+         }
+         LOGGER.info("Database is"+dbName);
+         RestWrapperOptions restWrapperOptions = null;
+         try {
+             DatabaseMetaData metaData = getHiveJDBCConnection(dbName).getMetaData();
+
+             ResultSet rs = metaData.getTables(null, null, null, null);
+             List<String> tables = new ArrayList<String>();
+             while (rs.next()) {
+                 String tableName = rs.getString("TABLE_NAME");
+                 tables.add(tableName.toUpperCase());
+             }
+
+             try {
+                 closeResultset(rs);
+                 closeConnection();
+             } catch (SQLException ex) {
+                 LOGGER.error("Error in close");
+             } finally {
+                 connection = null;
+             }
+             List<RestWrapperOptions.Option> options = new ArrayList<RestWrapperOptions.Option>();
+             for(String table : tables)
+             {
+                 RestWrapperOptions.Option option = new RestWrapperOptions.Option(table,table);
+                 options.add(option);
+             }
+             restWrapperOptions = new RestWrapperOptions(options, RestWrapperOptions.OK);
+         } catch (Exception e) {
+             LOGGER.error("error occured :" + e);
+             restWrapperOptions = new RestWrapperOptions(e.getMessage(), RestWrapperOptions.ERROR);
+         }
+         return restWrapperOptions;
+     }
+
+ */
+    @RequestMapping(value = "/createjobs/{checked}", method = RequestMethod.POST)
+    @ResponseBody
+    public RestWrapper createJob(@RequestParam Map<String, String> map, Principal principal, @PathVariable("checked") String[] checkedTables) {
+        LOGGER.debug(" value of map is " + map.size());
+        RestWrapper restWrapper = null;
+
+        String processName = null;
+        String processDesc = null;
+        Integer busDomainID = null;
+        Integer tablesSize = 0;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "tablesSize" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)